app/data_providers/slate_providers/new_tab_slate_provider.py

```diff
@@ -95,13 +104,35 @@ async def rank_corpus_items(self, items: List[CorpusItemModel], *args, **kwargs)
         metrics = await self.corpus_engagement_provider.get(
             self.recommendation_surface_id, self.configuration_id, items)
 
+        alpha_prior = 188  # beta * P99 German NewTab CTR for 2023-03-28 to 2023-04-05 (1.5%)
+        beta_prior = 12500  # 0.5% of median German NewTab item impressions for 2023-03-28 to 2023-04-05.
```

Comment:
These values might very well be sub-optimal. I'd like to keep changes to control (enable_ranking_by_region=false) out-of-scope for this PR. Please feel free to open a PR if anyone from ML has a better prior.